doc/usecases/elektra_web/UC_cluster_config.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Remote configuration of a cluster
+
+## Summary
+
+Title: Remote configuration of a cluster
+Scope: Configuration
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User configures a cluster (multiple instances at once) via the client.
+
+## Scenarios
+
+Precondition: Add cluster
+Main success scenario: User successfully changes configuration of the cluster.
+Alternative scenario: One of the instances is not online. The user is informed about the issue.
+Error scenario: Technical problems while persisting configuration to one of the instances. The user is informed about the issue.
```

Comment:
Again: What happens, are there any changes anywhere?

You could describe the behaviour also in "Postcondition".